### DISCUSSION TITLE
Removing all faces from a photo?

### DISCUSSION THREAD

**Discussion by matrixes** [immich-app/immich#12732] [Q&A]
I'm not sure if I'm missing something _very_ obvious, but having skimmed the documentation, FAQ and discussions I still can't figure out if I can remove all faces from a photo manually.

As it stands, all I can do is replace a face with another one, but not _remove_ all of the faces?

![image](https://github.com/user-attachments/assets/b15c8c84-147f-47f9-854b-743815b538aa)

Arguably, it _could_ be a face, but it's not. It's a pattern on a table :). I'd like to permanently remove the "face" tag but I can't figure out how to accomplish that.

**Comment by deurk** [comment]
I'd be interested in this as well

**Comment by ffchung** [comment]
I really need this feature. Remove face from photos.

**Reply by ffchung** [reply]
And need function "Remove face from the people"
